dir.props

```diff
@@ -56,8 +57,10 @@
     <GitVersioningDir>$(PackagesDir)\Nerdbank.GitVersioning\$(GitVersioningVersion)\build\</GitVersioningDir>
 
     <!-- Tell build tools to use the full framework dlls that contain build tasks (like the nuget assets task) -->
-    <BuildToolsTaskDir>$(ToolsDir)net45/</BuildToolsTaskDir>
-    <PackagingTaskDir>$(ToolsDir)net45/</PackagingTaskDir>
+    <BuildToolsTaskDir Condition="'$(MSBuildRuntimeType)'=='Full'">$(ToolsDir)net45\</BuildToolsTaskDir>
+    <BuildToolsTaskDir Condition="'$(MSBuildRuntimeType)'!='Full'">$(ToolsDir)</BuildToolsTaskDir>
```

Comment:
Does this work in our .net core builds because the bootstrap msbuild coming from build tools is so old that MSBuildRuntimeType is empty and therefore different from "Full"?

Reply:
We don't download MSBuild when on `Full` (machine install on the image). We only download it when on `Core`. So == Core would not work, but != Full should.